packages/core/package.json

```diff
@@ -35,6 +35,7 @@
     "@carbon/react": "^1.81.0",
     "@carbon/themes": "^11.50.0",
     "@carbon/type": "^11.38.0",
+    "@carbon/utilities-react": "0.7.0",
```

Comment:
We've refactored our storybook to now be located within `packages/ibm-products`, so you can remove this change.